packages/server/src/blitz-version.ts

```diff
@@ -0,0 +1,32 @@
+import {pathExists, readFile, writeFile} from "fs-extra"
+import {resolve} from "path"
+
+export function getBlitzVersion(): string | null {
+  try {
+    const path = require.resolve("blitz/package.json")
```

Comment:
Not sure if this is warranted but I am wondering if we should set an env var for this as the process launches? 🤔

Reply:
not sure what you mean, is it putting the `blitz version` in `process.env`?

Reply:
Just an idea for solving the issues with `require.resolve`. It might not work if env isn't passed down between contexts - although it might be fine. I would investigate if anything already exists first and if not I would test if setting an env var in `blitz` package becomes available to `@blitzjs/server`. If so then one option might be to set the version there as a global and use that. This may not be the best option. Tradeoff is that we then would be relying on remote part of the system. The other alternative I see here would be to manually look for the package.json and parse it with fs. There are possibly a bunch of other ways to do this too so feel free to think of something else.

Reply:
I think the `process.env` will be passed throughout the entire process (unless we do some multi-node processes which I am afraid `blitz` does not do it)

The parsing of package.json is the one that I have done (however still quite problematic using `require.resolve` because the module path needs to be resolved well). However, in my code, if `require.resolve` fails, I will return `null` value. This will ensure everything works well, however, this will lead to mismatch version between the cache and installed one and in extend will result to clean cache every `dev` command is called

Brute-force looking for `blitz` package.json might be able to fix this even if `require.resolve` fails.

Want me to try the `process.env` approach? @ryardley

Reply:
@ryardley require.resolve is working fine, the only issue is in our test environment.

Reply:
Then it makes sense to just fix that